test/uk/gov/hmrc/securemessage/services/SecureMessageServiceSpec.scala

```diff
@@ -45,25 +50,47 @@ class SecureMessageServiceSpec extends PlaySpec with ScalaFutures with MockitoSu
   implicit val mat: Materializer = NoMaterializer
   val listOfCoreConversation = List(
     ConversationUtil.getFullConversation("D-80542-20201120", "HMRC-CUS-ORG", "EORINumber", "GB1234567890"))
+  val cnvWithEmail = ConversationUtil.getConversationRequest(true).asConversation("cdcm", "123")
+  val cnvWithNoEmail: Conversation = ConversationUtil.getConversationRequest(false).asConversation("cdcm", "123")
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
   "createConversation" must {
 
-    "return CREATED (201) when an email address is provided" in new TestCase {
-      when(mockRepository.insertIfUnique(any[Conversation])(any[ExecutionContext])).thenReturn(Future(true))
-      private val result = service.createConversation(ConversationUtil.getConversationRequest(true), "cdcm", "123")
-      status(result) mustBe CREATED
+    s"return CREATED ($CREATED) when an email address is provided" in new TestCase {
+      reset(mockRepository)
```

Comment:
it is better to have mocks in test context, you don't need resets then. 
And good to be consistent, all mocks and test data should be in test context across all tests.